src/Plugins/DBFTPlugin/Consensus/ConsensusService.cs

```diff
@@ -85,29 +90,30 @@ private void InitializeConsensus(byte viewNumber)
                 Log($"View changed: view={viewNumber} primary={context.Validators[context.GetPrimaryIndex((byte)(viewNumber - 1u))]}", LogLevel.Warning);
             Log($"Initialize: height={context.Block.Index} view={viewNumber} index={context.MyIndex} role={(context.IsPrimary ? "Primary" : context.WatchOnly ? "WatchOnly" : "Backup")}");
             if (context.WatchOnly) return;
+            blockGenTime = neoSystem.GetBlockGenTime();
```

Comment:
You need to initialize it even on watch-only nodes, move this line before `if (context.WatchOnly) return;`.

Reply:
https://github.com/neo-project/neo/pull/3622#discussion_r2020964623 will solve the problem.

Reply:
Solved in [f167925](https://github.com/neo-project/neo/pull/3622/commits/f1679256226ee98a7e2476b7ec3bab23b3693270).